src/goto-instrument/nondet_static.cpp

```diff
@@ -38,10 +38,6 @@ void nondet_static(
       const symbol_exprt &sym=to_symbol_expr(
           to_code_assign(instruction.code).lhs());
 
-      // is it a __CPROVER_* variable?
```

Comment:
If we are going to switch from marking internal / CPROVER generated variables by a CPROVER_PREFIX and instead use this new comment attribute, please could we do it across the whole code-base as I doubt that nondet-static is the only place that makes this distinction.

Also, "change how we mark internal variables" is a bigger change that should be in a PR marked "Fix-up nondet-static".

Reply:
While I agree with what Martin said (that we should reconsider various `__CPROVER_` prefixed symbols): 1) for several of them the main purpose of the prefix is disambiguation (avoid conflicts with user-provided symbols; 2) let's not ask for huge extra work going well beyond the purpose of the PR (this is happening in several PRs and often stalls progress) - create an issue so that it isn't forgotten instead.